docs/proposals/20210628-workloadspread.md

```diff
@@ -0,0 +1,291 @@
+---
+title: WorkloadSpread
+authors:
+  - "@BoltsLei"
+reviewers:
+  - "@Fei-Guo"
+  - "@furykerry"
+  - "@FillZpp"
+creation-date: 2021-06-28
+last-updated: 2021-07-05
+status: implementable
+---
+
+# WorkloadSpread
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a proposal and for highlighting
+any additional information provided beyond the standard proposal template.
+[Tools for generating](https://github.com/ekalinin/github-markdown-toc) a table of contents from markdown are available.
+
+## Motivation
+
+Define a CRD named WorkloadSpread, which can support configuration and management for elastic and multi-domain applications,
+even more powerful and flexible than [UnitedDeployment](https://openkruise.io/en-us/docs/uniteddeployment.html) in some ways.
+
+Not like the workload type, WorkloadSpread can be used with those stateless workloads, such as CloneSet, Deployment and ReplicaSet.
```

Comment:
is it possible to support job workloads?

Reply:
In my opinion,  workload spread can also support the Job workload. We have considered the Pod status for 
`succeed` or `failed`.